spotify-backup.py

```diff
@@ -43,12 +42,15 @@ def get(self, url, params={}, tries=3):
 	
 	# The Spotify API breaks long lists into multiple pages. This method automatically
 	# fetches all pages and joins them, returning in a single list of objects.
-	def list(self, url, params={}):
-		response = self.get(url, params)
+	def list(self, url, params={}, root=''):
+		response = self.get(url, params, root=root)
 		items = response['items']
 		while response['next']:
```

Comment:
If we can tell how many pages we have in total, using `tqdm` might be nice (I don't think it'll work on a `while` loop though.